addons/misc/functions/fnc_checkFAKSlot.sqf

```diff
@@ -0,0 +1,3000 @@
+                        case Slot_1_2_3_5_7_8: { _hasTrueValue pushback true; };
+                        case Slot_1_2_3_6_7_8: { _hasTrueValue pushback true; };
+                        case Slot_1_2_4_5_6_7: { _hasTrueValue pushback false; };
+                        case Slot_1_2_4_5_6_8: { _hasTrueValue pushback true; };
+                    };
+                };
+
+                if (_x >= SlotsLeft_6D_2Min && _x <= SlotsLeft_6D_2Max) then {
+                    switch (_x) do {
+                        case Slot_1_2_4_5_7_8: { _hasTrueValue pushback true; };
+                        case Slot_1_2_4_6_7_8: { _hasTrueValue pushback true; };
+                        case Slot_1_2_5_6_7_8: { _hasTrueValue pushback true; };
+                        case Slot_1_3_4_5_6_7: { _hasTrueValue pushback false; };
+                        case Slot_1_3_4_5_6_8: { _hasTrueValue pushback true; };
+                        case Slot_1_3_4_5_7_8: { _hasTrueValue pushback true; };
+                        case Slot_1_3_4_6_7_8: { _hasTrueValue pushback true; };
+                        case Slot_1_3_5_6_7_8: { _hasTrueValue pushback true; };
+                        case Slot_1_4_5_6_7_8: { _hasTrueValue pushback true; };
+                    };
+                };
+
+                if (_x >= SlotsLeft_6D_3Min && _x <= SlotsLeft_6D_3Max) then {
+                    switch (_x) do {
+                        case Slot_2_3_4_5_6_7: { _hasTrueValue pushback false; };
+                        case Slot_2_3_4_5_6_8: { _hasTrueValue pushback true; };
+                        case Slot_2_3_4_5_7_8: { _hasTrueValue pushback true; };
+                        case Slot_2_3_4_6_7_8: { _hasTrueValue pushback true; };
+                        case Slot_2_3_5_6_7_8: { _hasTrueValue pushback true; };
+                        case Slot_2_4_5_6_7_8: { _hasTrueValue pushback true; };
+                        case Slot_3_4_5_6_7_8: { _hasTrueValue pushback true; };
+                    };
+                };
+            };
+
+            // 7 Slots left
+            if (_x >= SlotsLeft_7D_Min && _x <= SlotsLeft_7D_Max) then { 
+
+                switch (_x) do {
+                    case Slot_1_2_3_4_5_6_7: { _hasTrueValue pushback false; };
+                    case Slot_1_2_3_4_5_6_8: { _hasTrueValue pushback true; };
+                    case Slot_1_2_3_4_5_7_8: { _hasTrueValue pushback true; };
+                    case Slot_1_2_3_4_6_7_8: { _hasTrueValue pushback true; };
+                    case Slot_1_2_3_5_6_7_8: { _hasTrueValue pushback true; };
+                    case Slot_1_2_4_5_6_7_8: { _hasTrueValue pushback true; };
+                    case Slot_1_3_4_5_6_7_8: { _hasTrueValue pushback true; };
+                    case Slot_2_3_4_5_6_7_8: { _hasTrueValue pushback true; };
+                    case Slot_1_2_3_4_5_6_7_8: { _hasTrueValue pushback true; };
+                };
+            };
+        };
+    };
+
+} forEach _ammoCount;
```

Comment:
Don't copy paste
Redo

Reply:
It's not copy past!
Each value is customly set for value.